sound/soc/sof/intel/cnl.c

```diff
@@ -246,5 +246,7 @@ const struct sof_intel_dsp_desc cnl_chip_info = {
 	.ipc_ack_mask = CNL_DSP_REG_HIPCIDA_DONE,
 	.ipc_ctl = CNL_DSP_REG_HIPCCTL,
 	.rom_init_timeout	= 300,
+	.ssp_count = 8,
```

Comment:
certainly not 8. where did you read this?

Reply:
@plbossart yes, it is 8 for max, but maybe some ports reserved.
From cAVS1.8 HAS:

`I2SPC | 3 | I2S Port Count.Value of 0 – 8 is allowed`

and on our current WHL RVP, the I2SPC is 3, but there have base address from 0x10000 (I2S0) to 0x17000 (I2S7). The last 4 are reserved, we can write but will not take effect.

Reply:
So use 3 then...

Reply:
OK, thanks!